src/assets/db.json

```diff
@@ -346,5 +346,12 @@
    "logo": "https://raw.githubusercontent.com/2ndalpha/gasmask/master/Resources/Images/About%20Box%20Image.png",
    "description": "Gas Mask is a simple hosts file manager for macOS. It allows editing of host files and switching between them.",
    "categories": ["Hosts", "System"]
+  },
+  {
+   "id": 50,
+   "name": "Numi",
+   "logo": "https://numi.app/images/numi-icon@2x.png",
+   "description": "Numi is a handy calculator app for macOS. It allows you to describe tasks naturally and instantly get an accurate answer.",
+   "categories": ["calculator", "productivity", "macOS"]
   }
```

Comment:
Since the db is updated, we can update the new id to 55. Numi is used as numi in brew cask. You can lower the initial character.
```json
{
   "id": 55,
   "name": "numi",
   "logo": "https://numi.app/images/numi-icon@2x.png",
   "description": "Numi is a handy calculator app for macOS. It allows you to describe tasks naturally and instantly get an accurate answer.",
   "categories": ["calculator", "productivity", "macOS"]
  }
```